### PR TITLE
gomnd: new configuration

### PR DESCRIPTION
**Pull request by ldez**
I rewrote the `gomnd` configuration to be like the other linters configuration.
I added a compatibility section, so no breaking changes.

The old configuration and the new configuration cannot be used at the same time: if the old configuration is defined, only this configuration will be used.

### CODE REVIEW

File: pkg/golinters/gomnd.go [hunk 1]
```diff
@@ -8,20 +8,29 @@ import (
...
-	if cfg != nil {
-		linterCfg = cfg.LintersSettings.Gomnd.Settings
+
+	if settings != nil {
+		// TODO(ldez) For compatibility only, must be drop in v2.
+		if len(settings.Settings) > 0 {
+			linterCfg = settings.Settings
+		} else {
+			linterCfg = map[string]map[string]interface{}{
+				"mnd": {
```
Comment (butuzov) [inline pkg/golinters/gomnd.go:20]: did you mean `gomnd` ?

Comment (ldez) [reply pkg/golinters/gomnd.go:20]: The analyzer name is `mnd`, so it's the right value.

https://github.com/tommy-muehle/go-mnd/blob/c5e4f67f0b1b0a8114d42c8b95709f4e6dd30b37/analyzer.go#L19

Comment (butuzov) [reply pkg/golinters/gomnd.go]: I see error too. You are right.

**Review by butuzov — APPROVED** [review]

**Comment by butuzov** [comment]
It looks better in your PR.

**Pull request merged**